### ISSUE TITLE
Utilize Axom's export CMake targets

### ISSUE THREAD

**Issue by white238** [LLNL/spheral#135]
I was helping a user debug a build issue with Spheral and noticed that Spheral is not utilizing Axom's export CMake target logic which creates a CMake target, `axom`, that includes all necessary information for Axom and its dependencies.  Using this would guard you against any changes in Axom and be more robust.

We were able to change your build system locally to work by doing the following in `cmake/InstallTPLs.cmake`:

```
# comment out Spheral logic that creates a basic Axom BLT Registered Library (deprecated in BLT btw for BLT Imported Libraries)
#Spheral_Handle_TPL(axom spheral_depends)
find_package(axom REQUIRED
                       NO_DEFAULT_PATH 
                       PATHS ${AXOM_DIR}/lib/cmake)
list(APPEND spheral_blt_depends axom)      
```

**Comment by mdavis36** [comment]
Thanks @white238 I'd like to transfer to a find_package approach throughout spheral in the near future. Our custom package logic is an artifact of our deprecated TPL system where we had the option of building all of our TPLs from CMake.

**Comment by mdavis36** [comment]
This has been addressed in 2023.03.0
